vulture_os/toolkit/api_parser/gatewatcher_alerts/gatewatcher_alerts.py

```diff
@@ -132,7 +143,8 @@ def execute(self):
         # Writting may take some while, so refresh token in Redis
         self.update_lock()
         # increment by 1ms to avoid duplication of logs
-        self.frontend.last_api_call = to + timedelta(microseconds=1)
-        self.frontend.save()
+        if logs != []:
```

Comment:
Can be simplified
```suggestion
        if logs:
```
is also safer, as condition won't validate if `logs` is `None`

Reply:
> Can be simplified
> 
> is also safer, as condition won't validate if `logs` is `None`

To be coherent with other collectors, prefer to do : 
```if len(logs) > 0:```

Reply:
To satisfy everyone, you could do `if logs and len(logs) > 0:` !

Reply:
We don't do systematically `if len(logs) > 0:`
For example : cisco_duo and vadesecure_o365